@xen-orchestra/fs/src/local.js

```diff
@@ -18,7 +60,7 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   async _closeFile(fd) {
-    return fs.close(fd)
+    return fs.close(fd.fd)
```

Comment:
If this is an independent fix, it should really be in its own PR with its own unit test.

Because it can fix a lot of unrelated issues.